codegenerator/generatemodel/integration_test.go

```diff
@@ -200,37 +201,10 @@ func TestDefineTask(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Exception thrown generating temporary credentials!\n\n%s\n\n", err)
 	}
-	myQueue = queue.New(*tempCreds)
+	myQueue = queue.New(tempCreds)
 	_, cs, err = myQueue.CancelTask(taskId)
 	if err != nil {
 		t.Logf("Exception thrown cancelling task with temporary credentials!\n\n%s\n\n", err)
 		t.Fatalf("\n\n%s\n", cs.HttpRequest.Header)
 	}
 }
-
-// Checks whether two json []byte are equivalent (equal) by formatting/ordering
-// both of them consistently, and then comparing if formatted versions are
-// identical. Returns true/false together with formatted json, and any error.
-func jsonEqual(a []byte, b []byte) (bool, []byte, []byte, error) {
-	a_, err := formatJson(a)
-	if err != nil {
-		return false, nil, nil, err
-	}
-	b_, err := formatJson(b)
-	if err != nil {
-		return false, a_, nil, err
-	}
-	return string(a_) == string(b_), a_, b_, nil
-}
-
-// Takes json []byte input, unmarshals and then marshals, in order to get a
-// canonical representation of json (i.e. formatted with objects ordered).
-// Ugly and perhaps inefficient, but effective! :p
-func formatJson(a []byte) ([]byte, error) {
-	tmpObj := new(interface{})
-	err := json.Unmarshal(a, &tmpObj)
-	if err != nil {
-		return a, err
-	}
-	return json.MarshalIndent(&tmpObj, "", "  ")
-}
```

Comment:
These are the two functions I moved into their own `jsontest` package in the `taskcluster-base-go` project - hence why they are gone from here.